.golangci.yml

```diff
@@ -36,6 +36,7 @@ linters:
   - govet
   - importas
   - ineffassign
+  - loggercheck
```

Comment:
Validates for complete k/v pairs (i.e. we get a finding if we add an odd number of arguments)